### ISSUE TITLE
CAR BOOKING SYSTEM

### ISSUE THREAD

**Issue by DHANDAPANISATTANATHAN** [SSOMENSREPO/CAR-BOOKING-SYSTEM#1]
# TEAM MEMBER: SAFI & SO

https://docs.google.com/spreadsheets/d/1hb07Qv0CpMM6IHYij6ay8kxAxCQhzfL3IX5yOIQn6OQ/edit#gid=0

**Comment by DHANDAPANISATTANATHAN** [comment]
@safiyullahmohideen 

# 20/02/2016 - shld show all form design in dev

**Comment by DHANDAPANISATTANATHAN** [comment]
# FEED BACK ON 24/2/2016

1. Add header and footer in all page
2. car image shld fit with all device. do as responsive page
3. Home page show the car image slider
4.next page onwards show header and footer.
5.font not visible, also booking form not aligned.

**Comment by sophiyaB** [comment]
For SQL TEAM
Remove the password field from the USER_LOGIN_DETAILS table.
Insert new car class field in CAR_DETAILS table

**Comment by DHANDAPANISATTANATHAN** [comment]
# SQL TEAM,

# UNBLOCK SO

**Comment by RAJALAKSHMIRAJKUMAR** [comment]
REMOVED PWD IN USER_LOGIN_DETAILS TABLE
ADD CAR CLASS FIELD IN CAR_DETAILS TABLE

THESE CHANGES UPDATED IN DEVELOPMENT SCHEMA & UAT SCHEMA

GIT HUB LINK : https://github.com/SSOMENSREPO/CAR-BOOKING-SQL/releases

Status : **Done**

**Comment by sophiyaB** [comment]
For SQL Team
Insert two new field rate per day and per hour in car details table

**Comment by RAJALAKSHMIRAJKUMAR** [comment]
@so
COMPLETED LOAD IN DEV CHECK NW.

Status: **Done**

**Comment by safiyullahmohideen** [comment]
@RAJALAKSHMIRAJKUMAR 
Create Email template and create configuration for mail sending same as LMC.

**Comment by sophiyaB** [comment]
@RAJALAKSHMIRAJKUMAR 
Insert the password field in USER_LOGIN_DETAILS table
Insert the Error message

**Comment by RAJALAKSHMIRAJKUMAR** [comment]
Below changes i updated in dev nw u check
Created Email template and create configuration for mail sending same as LMC.
Insert the password field in USER_LOGIN_DETAILS table
Insert the Error message

git hub link : https://github.com/SSOMENSREPO/CAR-BOOKING-SQL/releases/tag/0.2

status : **Done**

**Comment by DHANDAPANISATTANATHAN** [comment]
# FEED BACK
fix the below
![issue_01032016](https://cloud.githubusercontent.com/assets/9882142/13428768/8f5755d8-dfe2-11e5-9efc-7aaa981fbd1a.png)

**Comment by DHANDAPANISATTANATHAN** [comment]
#  PUT ON HOLD IMMEDIATELY
